kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/authentication/JsonWebToken.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.kuksa.authentication
+
+/**
+ * A JsonWebToken can be used to authenticate against the DataBroker. For authentication to work the DataBroker must be
+ * started with authentication enabled first.
+ * The JsonWebToken is defined by an [authScheme] and [token]. The [authScheme] is set to "Bearer". The [token] should
+ * contain a valid JsonWebToken.
+ *
+ * It will be send to the DataBroker as part of the Header Metadata in the following format:
+ *
+ * Headers
```

Comment:
Empty lines are not recognized by kotlin doc. Maybe add "####" before Header so it looks a bit nicer.